packages/components/tile/src/components/osds-tile/osds-tile.scss

```diff
@@ -5,112 +5,114 @@
   }
-}
 
-:not([checking])[disabled] {
-  @include osds-tile-on-selected-host {
-    opacity: .5;
+  slot[name="end"] {
+    justify-content: flex-end;
   }
-}
 
-[hoverable]:not([disabled]) {
-  @include osds-tile-on-selected-host {
-    cursor: pointer;
+  &__text--center {
+    justify-content: center;
   }
-}
 
-[rounded] {
-  @include osds-tile-on-selected-host {
-    border-radius: var(--ods-size-border-radius-02);
+  :host(&) {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    border: solid var(--ods-size-inset-02);
   }
-}
 
-slot[name="start"],
-slot[name="end"],
-.tile__centered-text {
-  display: inline-flex;
-}
+  :host(&#{&}--checking){
+    @include osds-tile-on-selected-host {
+      position: relative;
+      overflow: visible;
+
+      &::after {
+        position: absolute;
+        top: -2px;
+        left: -2px;
+        z-index: -1;
+        background-size: 400%;
+        width: calc(100% + 4px);
+        height: calc(100% + 4px);
+        animation: steam 3s linear infinite;
+        content: '';
+        filter: blur(5px);
+      }
+
+      @keyframes steam {
+        0% {
+          background-position: 0 0;
+        }
 
-::slotted([slot="start"]),
-::slotted([slot="end"])  {
-  display: inherit;
+        50% {
+          background-position: 100% 0;
+        }
 
-  &:empty {
-    display: none;
+        100% {
+          background-position: 0 0;
+        }
+      }
+    }
   }
-}
 
-slot[name="start"] {
-  justify-content: flex-start;
-}
+  :host(&#{&}--disabled){
+    @include osds-tile-on-selected-host {
+      cursor: not-allowed;
 
-slot[name="end"] {
-  justify-content: flex-end;
-}
+      * {
+        pointer-events: none;
+      }
+    }
+  }
 
-.tile__centered-text {
-  justify-content: center;
-}
+  :host(&#{&}--disabled:not(&--checking)){
+    @include osds-tile-on-selected-host {
+      opacity: .5;
+    }
+  }
 
-[checking] {
-  @include osds-tile-on-selected-host {
-    position: relative;
-    overflow: visible;
-
-    &::after {
-      position: absolute;
-      top: -2px;
-      left: -2px;
-      z-index: -1;
-      background-size: 400%;
-      width: calc(100% + 4px);
-      height: calc(100% + 4px);
-      animation: steam 3s linear infinite;
-      content: '';
-      filter: blur(5px);
+  :host(&#{&}--hoverable:not(&--disabled)) {
+    @include osds-tile-on-selected-host {
+      cursor: pointer;
     }
+  }
 
-    @keyframes steam {
-      0% {
-        background-position: 0 0;
-      }
+  :host(&#{&}--inline){
+    display: inline-flex;
+    width: var(--width);
+  }
 
-      50% {
-        background-position: 100% 0;
-      }
+  :host(&:not(&--inline)){
+    @include osds-tile-on-selected-host {
+      display: grid;
+      grid-template-columns: auto 1fr auto;
+      width: 100%;
+    }
+  }
 
-      100% {
-        background-position: 0 0;
-      }
+  :host(&#{&}--rounded){
+    @include osds-tile-on-selected-host {
```

Comment:
Not sure but you have a double host(&) here

Reply:
Isn't this suppose to mean :
:host(.osds-tile.osds-tile--rounded) ?

Reply:
you don't need to duplicate the selector
`:host(.osds-tile--rounded)` is enough

Reply:
It's changed